tasks/broad/RNAWithUMIsTasks.wdl

```diff
@@ -101,6 +101,39 @@ task ExtractUMIs {
   }
 }
 
+# Adapter clipping
+task Fastp {
+  input {
+    File fastq1
+    File fastq2
+    String output_prefix
+    File adapter_fasta = "gs://broad-dsde-methods-takuto/RNA/resources/Illumina_adapters.fasta" # sato: move to a public place
```

Comment:
Is this something we may want to be configurable in the future i.e. as top level wdl input? Its not necessary just something to consider

Reply:
I won't change but we will probably need to move it somewhere.

@gbggrant where should it go? All the other resources files are in `gs://gcp-public-data--broad-references/` but it might not be appropriate there since it's not part of the reference.

Reply:
It's a good question - still will probably put it in the Google public references bucket.